### PR TITLE
Strengths weaknesses page

### PR DESCRIPTION
**Pull request by m-davoodi**

### CODE REVIEW

File: src/pages/strengths-weaknesses-page/StrengthsWeaknessesPage.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React, { useCallback } from 'react';
+import { Container } from '@material-ui/core';
+import { StrengthsWeaknessesForm } from './StrengthsWeaknessesForm';
```
Comment (sijav) [inline src/pages/strengths-weaknesses-page/StrengthsWeaknessesPage.tsx]: add index.ts in src/pages/strengths-weaknesses-page which exports at least StrengthsWeaknessesPage

File: src/pages/strengths-weaknesses-page/StrengthsWeaknessesPage.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import { useMutation } from 'src/relay';
+import graphql from 'babel-plugin-relay/macro';
+import { StrengthsWeaknessesPageMutation } from './__generated__/StrengthsWeaknessesPageMutation.graphql';
+import { useAuthGuardUser } from 'src/core/auth/UserContext';
```
Comment (sijav) [inline src/pages/strengths-weaknesses-page/StrengthsWeaknessesPage.tsx]: Order imports

File: src/pages/strengths-weaknesses-page/StrengthsWeaknessesForm.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { DictInput, DictInputItem, Forminator, SubmitButton } from 'src/shared/forminator';
+import StrengthsOrWeaknesses from 'src/shared/strengths-weaknesses/StrengthsOrWeaknesses';
```
Comment (sijav) [inline src/pages/strengths-weaknesses-page/StrengthsWeaknessesForm.tsx]: if #23 merged import from src/shared/strengths-weaknesses

File: src/pages/strengths-weaknesses-page/StrengthsWeaknessesPage.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      }
+    }
+  `);
+
+const transformData = (data: StrengthsWeaknessesFormData) => {
+  const strengths = data.strengths?.filter(val => val !== '');
+  const weaknesses = data.weaknesses?.filter(val => val !== '');
+
+  return {
+    strengths: strengths?.length ? strengths : null,
```
Comment (smmoosavi) [inline src/pages/strengths-weaknesses-page/StrengthsWeaknessesPage.tsx]: empty array not null

**Pull request merged**